packages/react-union/src/utils.js

```diff
@@ -41,6 +43,8 @@ export const createElementWithId = (id, parent, elementType = 'div') => {
 	return element ? element : createElement(id, parent, elementType);
 };
 
+export const getDisplayName = Component => Component.displayName || Component.name || 'Component';
+
 export const noop = always(null);
 
-export const getDisplayName = Component => Component.displayName || Component.name || 'Component';
+export const mergeDeepRightAll = reduce(mergeDeepRight, {});
```

Comment:
This could probably be imported from ramda-extension, but it's not included in the project and I'm unsure if you want to include it in the react-union package just for this one import.